src/GapsInCareHelpers.ts

```diff
@@ -344,6 +352,26 @@ export function calculateReasonDetail(
                 }
               });
             }
+          } else if (f.type === 'notnull') {
+            const notNullFilter = f as NotNullFilter;
+            const resources = detailedResult.clauseResults?.find(
+              cr => cr.libraryName === r.libraryName && cr.localId === r.retrieveLocalId
+            );
+            const attr_path = notNullFilter.attribute.split('.');
+            if (resources) {
+              // Access desired property of FHIRObject
+              resources.raw.forEach((r: any) => {
+                let desiredAttr = r;
+                attr_path.forEach(key => {
```

Comment:
```suggestion
            const attrPath = notNullFilter.attribute.split('.');
            if (resources) {
              // Access desired property of FHIRObject
              resources.raw.forEach((r: any) => {
                let desiredAttr = r;
                attrPath.forEach(key => {
```

sorry

Reply:
ugh- this was a quick change because vscode was doing something *dumb* with debugging, and I forgot to change it back. Sorry my rubyisms are trying to infect the js.